.gitignore

```diff
@@ -10,6 +10,7 @@ Session.vim
 
 __pycache__
 .*.swp
-
+venv
```

Comment:
The convention I follow locally is `.venv`, but I'm happy to include this (which I guess is what's listed in the docs?).

Reply:
Ah sure, that makes sense. The docs currently specify:
```
virtualenv -p python3 venv
```

Rather than:
```
virtualenv -p python3 .venv
```

I think it makes sense to just update the docs to specify `.venv`